libs/datasets/sources/hsa_population.py

```diff
@@ -40,6 +42,12 @@ def make_dataset(cls) -> timeseries.MultiRegionDataset:
         populations = FIPSPopulation.make_dataset().static
         county_index = populations.index.map(get_location_level) == AggregationLevel.COUNTY
         counties = populations[county_index].copy()  # copy to remove SettingWithCopy error
+        hsa_names: pd.DataFrame = (
+            pd.read_csv(HSA_LIST_PATH, dtype={CommonFields.HSA: str})
+            .loc[:, [CommonFields.HSA, CommonFields.HSA_NAME]]
+            .drop_duplicates()
+            .assign(hsa_name=lambda row: row[CommonFields.HSA_NAME].fillna("Unknown HSA Name"))
```

Comment:
Can you make this just "Unknown"?

I'm imagining in the frontend we're going to have copy something like:
> Over the past 7 days, the {hsaName} Health Service Area had xyz staffed beds and xyz covid patients, resulting in xy% of staffed beds filled with covid patients.